src/testing/taylor_green_vortex_restart_check.cpp

```diff
@@ -80,12 +82,20 @@ int TaylorGreenVortexRestartCheck<dim, nstate>::run_test() const
     const int initial_iteration_restart = restart_file_index; // assumes output mod for restart files is 1
     const double time_at_which_the_run_is_complete = this->all_parameters->flow_solver_param.final_time;
     Parameters::AllParameters params_incomplete_run = reinit_params(true,false,time_at_which_we_stop_the_run);
-    Parameters::AllParameters params_restart_to_complete_run = reinit_params(false,true,time_at_which_the_run_is_complete,time_at_which_we_stop_the_run,initial_iteration_restart,0.0,restart_file_index);
 
     // Integrate to time at which we stop the run
     std::unique_ptr<FlowSolver::FlowSolver<dim,nstate>> flow_solver_incomplete_run = FlowSolver::FlowSolverFactory<dim,nstate>::select_flow_case(&params_incomplete_run, parameter_handler);
     static_cast<void>(flow_solver_incomplete_run->run());
 
+    const double time_step_at_stop_time = flow_solver_incomplete_run->flow_solver_case->get_constant_time_step(flow_solver_incomplete_run->dg);
+    Parameters::AllParameters params_restart_to_complete_run = reinit_params(false,true,
+                                                                             time_at_which_the_run_is_complete,
+                                                                             time_at_which_we_stop_the_run,
+                                                                             initial_iteration_restart,
+                                                                             0.0,
```

Comment:
Use a variable name or add comment - with so many inputs it's hard to remember which this is.